### PR TITLE
Change export script to use hqmf_set_id

### PR DESCRIPTION
**Pull request by alexanderelliott121**
Changes the export script to use the hqmf_set_id

### CODE REVIEW

**Review by edeyoung — DISMISSED** [review]

**Review by mulcahyk — COMMENTED** [review]
Consider reworded this error message.  "measure_id" for me in this context is ambiguous.  There are four possible identifiers for "measure_id": the measure_id in the database, cms is, hqmf id, and the hqmf set id.  Since the message is dynamic you could say "the CMS ID" or the "HQMF set ID" doesn't match.

File: lib/tasks/generate_cql_fixtures.rake [hunk 1]
```diff
@@ -145,7 +145,7 @@ namespace :bonnie do
       CqlMeasure.by_user(user).each do |measure|
         if (cms_hqmf.downcase  == 'cms' && measure.cms_id == measure_id)
```
Comment (mulcahyk) [inline lib/tasks/generate_cql_fixtures.rake]: Mongoid is case-sensitive.  Consider making this equality and the following one case-insensitive; CMS113**v**6 does not match CMS113**V**6.  Or if you don't want to do that update the error message to let the user know that they may want to double check the case of the text in their arguments.

-- This comment applies to every instance where you are relying on user input.

**Comment by alexanderelliott121** [comment]
comparison set to use downcase, error message updated.

**Comment by mulcahyk** [comment]
The changes look good to me.  Tested and they function as expected.

**Review by mulcahyk — APPROVED** [review]

**Review by edeyoung — APPROVED** [review]

**Pull request merged**